src/wazuh_db/wdb_upgrade.c

```diff
@@ -112,6 +113,16 @@ wdb_t * wdb_upgrade_global(wdb_t *wdb) {
         }
     }
 
+    char output[OS_MAXSTR + 1] = {0};
+
+    if (version < (int)(sizeof(UPDATES)/sizeof(char*))) {
+        if (OS_SUCCESS == wdb_global_create_backup(wdb, output)) {
+            mdebug1("Create pre-upgrade global DB snapshot %s", output+3);
+            post_upgrade = TRUE;
+            sleep(1);
```

Comment:
We could avoid this `sleep()` if we improve the backup command with an extra argument to add more information to the backup name

Reply:
A solution is implemented, please take a look. Thanks!